tools/rust-lint

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Run Rust lints: fmt, clippy and udeps.
+
+set -e
+
+pushd rust
+
+echo "Check code formatting"
+cargo fmt --check
+
+echo "Check Clippy warnings"
+cargo clippy -- -D warnings
```

Comment:
where do we define the clippy warnings checks?

Reply:
We check the clippy warnings on the CI here
https://github.com/trustwallet/wallet-core/blob/826174f0c448f1de38646cdca0725be35da9c0ac/.github/workflows/linux-ci-rust.yml#L43-L45
Could you please clarify what you mean if this is not an answer to you question?

Reply:
I mean how do you define specific warnings for clippy? is that just the default one?